File: Java/src/eu/extrac/harry/cleanflight/SensorData.java
package eu.extrac.harry.cleanflight;

public class SensorData 
{

	private Vector3Float accelerometer;
	private Vector3Float magetometer;
	private Vector3Float gyroscope;
	private Vector3Float kinematics;
	
	
	public SensorData()
	{
		accelerometer = new Vector3Float();
		magetometer = new Vector3Float();
		gyroscope = new Vector3Float();
		kinematics = new Vector3Float();
	}
	
	
	public Vector3Float getAccelerometer() { return accelerometer; }
	public Vector3Float getMagetometer() { return magetometer; }
	public Vector3Float getGyroscope() { return gyroscope; }
	public Vector3Float getKinematics() { return kinematics; }	
}

File: C#/CleanFlight/SensorData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CleanFlight
{
    public class SensorData
    {

        public SensorData()
        {
            Gyroscope = new Vector3<float>();
            Accelerometer = new Vector3<float>();
            Magetometer = new Vector3<float>();
            Altitide = 0;
            Sonar = 0;
            Kinematics = new Vector3<float>();
        }


        public Vector3<float> Gyroscope { get; internal set; }

        public Vector3<float> Accelerometer { get; internal set; }

        public Vector3<float> Magetometer { get; internal set; }

        public float Altitide { get; internal set; }

        public float Sonar { get; internal set; }

        /// <summary>
        /// X => Angle X (degrees)
        /// Y => Angle Y (degrees)
        /// Z => Heading (degrees)
        /// </summary>
        public Vector3<float> Kinematics { get; internal set; }

    }
}
